### DISCUSSION TITLE
[Inspiration] Species (bi-peds)

### DISCUSSION THREAD

**Discussion by ZachZent** [tsunamayo/Starship-EVO#4517] [Suggestions Done]
Now that we have seen the player model and the information from the Q&A, I'm sure you are now looking to the future. I know you want to keep it bipedal for simplicity and animation sake, but I hope you are or will consider more then just humanoids. There are many types of bipeds (or at least their heads) which can make the galaxy seem more then just humans and humans with pointy ears.

![Uploading Jr1DxpP - Imgur.jpg…]()
This complication of images is many of the races from the game Stellaris with the non-bipedal ones removed. From left to right/top down: humanoid, mammalian, reptilian, avian, arthropoid, molluscoid, fungoid, plantoid (yes... evolved plants...) and machine. Not that you will be adding this many individual races, but I hope that you will be inspired by a few of them at the very least. Enough to think of some creatures beyond the skinned and eared. Adds quite a bit of diversity to the galaxy

**Comment by ZachZent** [comment]
Not sure if the picture is working so here is a link
https://imgur.com/Jr1DxpP

**Comment by ultrak2k** [comment]
If you dont add any of those atleast add catgirls pls tsuna

**Comment by tsunamayo** [comment]
hehe, yes I want to go into the funky vibe. Skeleton / dimension wise it would stay close to human for budget reason though. Here are a few concept made by my concept artist, tell me if you like some of them. I want to keep that simple comic look, as opposed to Stelaris.
I will have my character artist work on some, but it will depend on the next funding wave.

https://imgur.com/a/uc5ljli

**Comment by ZachZent** [comment]
They are exactly what I was hoping for. Too many people when making aliens just go for the basics. I am glad you are moving with some more exotic aliens and already have concept art. Hopefully some of the pics I shared will inspire a few more as most of the concept art seems to only be plantoids and a few mammalians. The ones I posted all have exchangeable uniforms/heads which makes making them easier. Past that, I wouln't forgot some of the more basic human types, but either way I am happy
